development/_static/css/phpbb.css

```diff
@@ -0,0 +1,24 @@
+.wy-side-nav-search {
+    background-color: #009BDF;
+}
+
+.wy-side-nav-search>div.version {
+    color: rgba(255, 255, 255, 0.7);
+}
+
+.wy-nav-content {
+    max-width: none; /* use max-width: 800px for button navigation, not content */
+}
+
+.wy-table-responsive table td,
+.wy-table-responsive table th {
+    white-space: normal;
+}
+
+.rst-footer-buttons {
+    max-width: 800px;
+}
+
+.rst-versions.shift-up  {
+    overflow: auto;
+}
```

Comment:
Shouldn't this CSS be from the other PR, it also seems a little different than in the other PR.

Reply:
This is the same CSS as before the table changes. I don't think it makes a lot of sense to port these to 3.2.x. Merging into 3.3.x and above, this should be resolved. The only addition is the rule for `.rst-versions.shift-up`.